src/main/resources/tags/pressure_plates.json

```diff
@@ -5,7 +5,7 @@
 		"minecraft:light_weighted_pressure_plate",
 		"minecraft:heavy_weighted_pressure_plate",
 		{
-			"id" : "minecraft:polished_blackstone_pressure_plate",
```

Comment:
blackstone pressure plat was added in 1.16
the tag for pressure plates was added in 1.16
so thats why i change that out